.pre-commit-config.yaml

```diff
@@ -34,7 +34,7 @@ repos:
         additional_dependencies:
           - flake8-print
         args:
-          - "--select=T201,F401,F402,F403"
+          - "--select=F401,F402,F403"
```

Comment:
This is only temporary for the prints, once I remove the prints, I'll remove this as well.